src/Neo.CLI/CLI/MainService.Tools.cs

```diff
@@ -27,18 +36,29 @@ partial class MainService
         [ConsoleCommand("parse", Category = "Base Commands", Description = "Parse a value to its possible conversions.")]
         private void OnParseCommand(string value)
         {
+            value = Base64Fixed(value);
```

Comment:
Why allow wrong formats? it's not a base64 string

Reply:
The `ClearHexString` method you wrote also fixes the unstandardized hexstring format. In addition there may be older versions of the node that generate base64 strings containing Unicode.